dev-portal/scripts/write-config.js

```diff
@@ -0,0 +1,30 @@
+const util = require('util');
+const writeFile = util.promisify(require('fs').writeFile)
+
+const { execute, r } = require('./utils.js')
+
+const buildConfig = require('../deployer.config.js')
+let stackName = buildConfig.stackName
+
+function writeConfig (swallowOutput) {
+  return execute(`aws cloudformation describe-stacks --stack-name ${stackName}`, swallowOutput)
+  .then((result) => {
+    let configData = {}
+
+    JSON.parse(result.stdout).Stacks[0].Outputs.forEach(output => {
```

Comment:
If you want to auto-detect other things (bucket names, etc), we can pretty easily make them available as an output so they can be pulled in here.

Reply:
Just have to edit template.yaml, and for each thing we want to expose, add it to outputs. The only thing we can't know in the CFN stack is the first bucket's name...